packages/ui/src/components/Modal/__stories__/Toggle.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Story } from '@storybook/react'
+import { useState } from 'react'
+import { Modal } from '..'
+import { Toggle } from '../../Toggle'
+
+export const ToggleStory: Story = props => {
+  const [toggled, setToggled] = useState(false)
+
+  return (
+    <Modal
+      {...props}
+      disclosure={dialog => (
+        <Toggle
+          name="toggle"
+          checked={toggled}
+          onChange={() => {
+            dialog?.toggle?.()
+            setToggled(true)
+          }}
+        />
+      )}
+      onBeforeClose={() => {
+        setToggled(false)
+      }}
+    >
+      <div>Content should be present in center of the modal</div>
+    </Modal>
+  )
+}
```

Comment:
<img width="1035" alt="Screenshot 2023-03-08 at 14 16 33" src="https://user-images.githubusercontent.com/15812968/223723057-f9851131-2cc3-42b8-b428-1cede81f8f67.png">

The code snippet of this story is not describing correctly what is happening, we missing the content of the disclosure. To display it in a better view you can remove the spread of props which will make storybook display the snippet exactly like we wrote it.

To do so you can:

```suggestion
export const ToggleStory: Story = () => {
  const [toggled, setToggled] = useState(false)

  return (
    <Modal
      disclosure={dialog => (
        <Toggle
          name="toggle"
          checked={toggled}
          onChange={() => {
            dialog?.toggle?.()
            setToggled(true)
          }}
        />
      )}
      onBeforeClose={() => {
        setToggled(false)
      }}
    >
      <div>Content should be present in center of the modal</div>
    </Modal>
  )
}
```